doc/rtd/development/summit.rst

```diff
@@ -0,0 +1,24 @@
+The cloud-init summit
+*********************
+
+One of the major highlights in our calendar is the cloud-init summit! The
+summit is an annual gathering of cloud-init contributors and community members
+held in Seattle, Washington.
+
+At the summit, we enjoy meeting with our fellow contributors to cloud-init,
+demoing recent developments, collecting feedback, and holding workshops to
+discuss outstanding issues, bugs, and possible fixes.
+
+After an unfortunate hiatus of a couple of years due to "global travel
+difficulties", we are pleased to announced that the next summit will be held in
+August 2023! More details to follow...
+
+Previous summits
+================
+
+.. toctree::
+   :maxdepth: 1
+
+   cloud-init: Summit 2017 <summit/2017_summit.rst>
+   summit/2018_summit.rst
+   summit/2019_summit.rst
```

Comment:
I think it's worth is adding a link to the online summit docs too for reference as well https://discourse.ubuntu.com/t/cloud-init-online-summit-2020/17867

Reply:
It would be a bit odd having a link to that pre-event page sitting alongside the post-event writeups that Josh did. Should we have a writeup of the virtual one?

Reply:
Good pt. Let's leave it out for now and can circle back next week to see if it merits a write up.